orsopy/fileio/base.py

```diff
@@ -585,6 +605,62 @@ class Column(Header):
     yaml_representer = Header.yaml_representer_compact
 
 
+@orsodataclass
+class ErrorColumn(Header):
+    """
+    Information about a data column.
+    """
+
+    error_of: str
+    error_type: Optional[Literal["uncertainty", "resolution"]] = None
+    distribution: Optional[
+        Tuple[Literal["gaussian", "triangular", "uniform", "lorentzian"], Literal["sigma", "FWHM"]]
```

Comment:
Grouping the distribution type with the width specifier here seems unecessary.  Why not split out the "sigma" vs "FWHM" to another attribute?

Reply:
Yes, good point. We separated it into "distribution" and "value_is" attributes.